src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php

```diff
@@ -40,43 +40,77 @@ public function testDebugSingleFormType()
     }
 
     /**
+     * @expectedException \Symfony\Component\Console\Exception\InvalidArgumentException
+     * @expectedExceptionMessage Could not find type "NonExistentType"
+     */
+    public function testDebugSingleFormTypeNotFound()
+    {
+        $tester = $this->createCommandTester();
+        $tester->execute(array('class' => 'NonExistentType'), array('decorated' => false, 'interactive' => false));
+    }
+
+    public function testDebugAmbiguousFormType()
+    {
+        $expectedMessage = <<<TXT
+The type "AmbiguousType" is ambiguous.
+Did you mean one of these?
```

Comment:
Even though I expected a line break "\n" before this line, https://github.com/symfony/symfony/pull/24219 has not yet been merged in `3.4`, no?